### ISSUE TITLE
F-15D & F-16D for USA missing in DB494?

### ISSUE THREAD

**Issue by Huib18** [PygmalionOfCyprus/cmo-db-requests#2698]
### DB Selector

DB3K

### Affected DBID(s)

F-16D & F-15D for United States

### Description of Problem

In the DB viewer of DB497 the F-16D & F-15D variants are missing for the USA.
Other countries, such as Greece, Turkey Israel are visible, and selectable for the F-16D.
Other countries such as Israel, Saudi Arabia and Japan are slectable for the F15D.
.

**Comment by Huib18** [comment]
It stops at Turkey:
![Clipboard02](https://user-images.githubusercontent.com/60408110/208986650-c81bcaec-4ee3-4a4c-905b-ec0e583ca744.jpg)

**Comment by PygmalionOfCyprus** [comment]
I just want to confirm the issue here, was there previously an F-16D / F-15D variant for the US that is now no longer there, or are you wondering why these aircraft are not also listed for the USA? I went back to DB 493 and I can't find any US versions in the DB. The D variants are both two-seat trainers which historically have been pretty unsung in the DB so it's wholly possible they were just never added for the USA.

**Comment by Huib18** [comment]
I could not select any F-16D for specific the United States. only F-16C is present in the DB
If I select another country like Turkey or Israel or Greece the F-16C & D is present for those countries.

The only impact is probably fuel wise, that their range is more limited than a one-seater. And every squadron has twin-seaters so this effects the flightplanning.

But they are present in the US ;-) And probably easily copied from F-16D's from other countries?
![Clipboard01](https://user-images.githubusercontent.com/60408110/211417860-326378eb-9e9f-4931-8f84-e234f61a2d66.jpg)

**Comment by PygmalionOfCyprus** [comment]
Okay, so this is a copy request to add F-16/F-15Ds to the USA, not a bug where a unit once present suddenly disappeared in the DB?

**Comment by Huib18** [comment]
Correct,  I was unaware that there were never USAF F-15/6 two-seaters in
the DB.

Thanks for your sharpness

**Comment by PygmalionOfCyprus** [comment]
The F-15Ds were added during the USAF review.

As for the F-16Ds...honestly, I don't want to. The F-16 family is currently by FAR the most agonizing aircraft family to maintain, given all the variants, concurrent modernization programs, etc. There are currently _90_ DB entries just covering US F-16s. Doubling this just to add identical entries with a (currently not used in-sim) 2x pilot number isn't worth the added overhead.

If in the future we implement OODA improvements or other gameplay changes for multi-pilot aircraft -- which I do want to do -- we'll revisit this. Or maybe when we implement dynamic loadouts and I write some helper scripts to make maintaining large aircraft families less horrific.

So, wontfix for now.
